Html

javitecsis opened the pull request:
Avance

javitecsis committed:
Html

Avance

javitecsis committed:
Create CSS Platzi GAme

teffcode reviewed (commented):
Great Job 🚀

Average:

HTML: 20%
CSS: 20%
JS: 0%

Things we can improve:

- Be more consistent with the whitespaces between HTML Tags.

javitecsis committed:
challenge-prework-frontend (v2)

Se realiza actiulaizacion de acuerdo a direccionamiento de coach y al entrenamiento personal desarrolado en la semana.

javitecsis committed:
challenge-prework-frontend

Actualizacion semana tres, vamos avanzando

javitecsis committed:
desafío-trabajo-previo-frontend (V2)

Se anexa CSS y HTML actualizado

javitecsis committed:
html

html

javitecsis commented:
Se anexa Actualizacion, seguimos avanzando
